latest/schema.json

```diff
@@ -155,7 +152,7 @@
                 "rule": {
                     "description": "Rule to be met. This rule can be one of the rules defined in 'rules' or it can be an inline rule.",
                     "type": "string",
-                    "pattern": "^[_a-zA-Z][a-zA-Z0-9_-]*$"
+                    "pattern": "([a-zA-Z][a-zA-Z0-9-_]*)|(^!?\\$[a-zA-Z_][a-zA-Z0-9_-]*\\((.|\\s)*(,(.|\\s)*)*\\)$)?"
```

Comment:
This regex has a limitation which is it will wrongly validate, for example, `$rule("touches-license") && !isElementOf($author(), $group("owners"))`, where there is a missing `$` on the `isElementOf` built-in call.